google/cloud/storage/internal/read_object_range_request.cc

```diff
@@ -76,6 +84,12 @@ ReadObjectRangeResponse ReadObjectRangeResponse::FromHttpResponse(
                                  last_byte, object_size};
 }
 
+std::ostream& operator<<(std::ostream& os, ReadObjectRangeResponse const& r) {
+  // TODO(coryan) - remember to print the contents before merging.
```

Comment:
What's the policy on TODO's for this code?
Two reasons for the question:

1) This TODO leaks your ldap, which Google Open Source policy generally prohibits (or at least discourages).
2) The GCS team switched to TODO(b/#) instead of TODO(ldap) to make TODO's trackable and force them to be resolved when closing bugs (we have job that re-opens bugs if it finds them closed with TODO(b/#) in the source). Could we do something similar here?

Reply:
Good catch.  We use `TODO(#NNN)` where NNN is a GitHub issue.  `coryan` is my GitHub username, not too worried about it leaking, it is all over the commit logs already.
Anyway, I need to fix the code here.  It should print the contents first.

Reply:
Fixed.  I refactored the existing code to dump binary data to the debug/info logs.